python/ray/rllib/agents/ppo/appo.py

```diff
@@ -51,8 +58,44 @@
 # __sphinx_doc_end__
 # yapf: enable
 
+def make_aggregators_and_optimizer(workers, config):
+    if config["num_aggregation_workers"] > 0:
+        # Create co-located aggregator actors first for placement pref
+        aggregators = TreeAggregator.precreate_aggregators(
+            config["num_aggregation_workers"])
+    else:
+        aggregators = None
+    workers.add_workers(config["num_workers"])
+    old_worker = workers._make_worker(

```

Comment:
@michaelzhiluo 	why do we need this?

Reply:
This is for importance sampling later on

Reply:
can this simply be `use_importance_sampling=True` for AsyncSamplesOptimizer? And then create the worker at a lower level? Also, why create a worker, why not create just a policy?